pkg/search/proxy/store/store.go

```diff
@@ -80,6 +80,9 @@ func (s *store) List(ctx context.Context, key string, opts storage.ListOptions,
 	}
 
 	options := convertToMetaV1ListOptions(opts)
+	if options.ResourceVersion == "" {
+		options.ResourceVersion = "0"
```

Comment:
in our cluster, when set options.ResourceVerion to 0, the karmada search pod will be OOM.